src/lit-element.ts

```diff
@@ -288,11 +302,15 @@ export class LitElement extends UpdatingElement {
   }
 
   connectedCallback() {
-    super.connectedCallback();
-    // Note, first update/render handles styleElement so we only call this if
-    // connected after first update.
-    if (this.hasUpdated && window.ShadyCSS !== undefined) {
-      window.ShadyCSS.styleElement(this);
+    if (this.hasAttribute('defer-hydration') && this._needsHydration) {
+      // Wait until parent hydrates
+    } else {
+      super.connectedCallback();
```

Comment:
I feel a little strange about deferring a super call. I wonder if this initialization logic can move into a method called conditionally by connectedCallback. ie rename enableUpdates to something like bootInstance and call that from connectedCallback and attributeChangedCallback. Super small nit though...

Reply:
All `super.connectedCallback()` does in UpdatingElement is call `this.enableUpdating()`.  I made the change to just call that method directly rather than `super.connectedCallback()`. That maybe looks less weird, but not sure if it's an improvement since the implementation is now disconnected from `UpdatingElement:connectedCallback`.